server/belga/io/feed_parsers/belga_dpa_newsml_2_0.py

```diff
@@ -101,6 +101,16 @@ def parse(self, xml, provider=None):
                     # Slugline and keywords is epmty
                     item['slugline'] = None
                     item['keywords'] = []
+                    genre_qcode = item_tree.xpath("//iptc:genre", namespaces=NS)[0].get('qcode')
+                    genre_role = item_tree.xpath("//iptc:genre/iptc:name", namespaces=NS)[0].get('role')
+                    headline = item['headline']
+                    if len(item['genre']) > 0:
```

Comment:
what about something like:
```py
genre_names = item_tree.xpath("//iptc:genre/iptc:name", namespaces=NS)
for genre_name in genre_names:
    if genre_name.get('role') == 'nrol:display':
        item["headline"] = "({genre}): {headline}".format(genre=genre_name.text, headline=item["headline"])
        break
```
I'm not sure if doing something with the `item["genre"]` is needed, nor if the first genre would have the role